lib/ansible/modules/cloud/huawei/hwc_network_vpc.py

```diff
@@ -0,0 +1,511 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 Huawei
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file at
+#     https://www.github.com/huaweicloud/magic-modules
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+###############################################################################
+# Documentation
+###############################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: hwc_network_vpc
+description:
+    - Represents an vpc resource.
+short_description: Creates a Huawei Cloud VPC
+version_added: 2.8
+author: Huawei Inc. (@huaweicloud)
+requirements:
+    - requests >= 2.18.4
+    - keystoneauth1 >= 3.6.0
+options:
+    state:
+        description:
+            - Whether the given object should exist in GCP
+        choices: ['present', 'absent']
```

Comment:
You should always include `type`. The order should always be `description`, `type`, `required`, `choices`, `default`, `added_in_version`.

Reply:
Done